manifests/init.pp

```diff
@@ -132,6 +135,7 @@
   Hash    $inputs                                = $telegraf::params::inputs,
   Hash    $outputs                               = $telegraf::params::outputs,
   Hash    $global_tags                           = $telegraf::params::global_tags,
+  Hash    $processors                            = $telegraf::params::processors,
```

Comment:
please put the default value here, and not into params.pp (that enables puppet-strings to pick it up)